.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
This is added by mistake, removed here: https://github.com/aws/aws-sdk-js-v3/pull/1350/commits/716b2899c852c8f887584ef560f50d737b395022